src/sqlancer/postgres/gen/PostgresExpressionGenerator.java

```diff
@@ -667,7 +714,21 @@ public PostgresExpression generateBooleanExpression() {
 
     @Override
     public PostgresSelect generateSelect() {
-        return new PostgresSelect();
+        PostgresSelect select = new PostgresSelect();
+        
+        // Add window functions to fetch columns if appropriate
+        if (Randomly.getBoolean()) {
```

Comment:
Could we set a lower probability for this? I remember that we have a method in the `Randomly` class for that.